templates/java/org/prism/Loader.java.erb

```diff
@@ -310,6 +310,18 @@ public class Loader {
         return (short) flags;
     }
 
+    private VariableInteger loadVariableInteger() {
```

Comment:
Minor: for the method name, "variable integer" seems a bit confusing with varint, which has a specific separate meaning (at least in how it's used here, like protobuf varint). GMP calls it "arbitrary precision integer". Not sure if there is a better term. "loadBigInteger" would be not ideal because it can be "small" too.